interpreters/toywasm/Makefile

```diff
@@ -51,21 +53,17 @@ CSRCS += vec.c
 CSRCS += wasi.c
 CSRCS += xlog.c
 
-CFLAGS += -DTOYWASM_USE_SEPARATE_EXECUTE
-CFLAGS += -DTOYWASM_USE_TAILCALL
-CFLAGS += -DTOYWASM_USE_JUMP_BINARY_SEARCH
-CFLAGS += -DTOYWASM_JUMP_CACHE2_SIZE=4
-CFLAGS += -DTOYWASM_USE_SEPARATE_LOCALS
-CFLAGS += -DTOYWASM_USE_SMALL_CELLS
-CFLAGS += -DTOYWASM_ENABLE_WRITER
+CFLAGS += ${shell $(INCDIR) "$(CC)" $(APPDIR)/interpreters/toywasm/include}
```

Comment:
Shouldn't we use `INCDIR_PREFIX` instead? Ref https://github.com/apache/incubator-nuttx-apps/pull/1403